velox/functions/sparksql/tests/ComparisonsTest.cpp

```diff
@@ -164,6 +164,12 @@ TEST_F(ComparisonsTest, testdecimal) {
       fmt::format("{}(c1, c0)", "lessthanorequal"),
       longDecimalsInputs,
       expectedGteLte);
+  auto expectedEqNullSafe =
+      makeNullableFlatVector<bool>({false, false, true, false, true, false});
```

Comment:
There's no null value, so makeFlatVector could be enough.

Reply:
Updated, thanks.